Data/pubtator_to_xml.py

```diff
@@ -0,0 +1,152 @@
+import gzip
+
+import tqdm
+
+
+def generate_annotation(file_object, tagged_term, index):
```

Comment:
It's `BioCWriter.iterfragments()`. Iterates over chunks of serialised XML fragments:
```python
with open(outfile, 'wb') as f:  # XML is a binary format
    for frag in writer.iterfragments():
        f.write(frag)
```